content/docs/command-reference/run.md

```diff
@@ -107,9 +107,10 @@ Relevant notes:
   defined as outputs every time its executed by DVC.
 
 - In some situations we have executed a stage and later notice that some of the
-  output files or directories it creates, which are already in the workspace,
-  are missing from `dvc.yaml` (`outs` field). We can
-  [add missing outputs to an existing stage](/docs/user-guide/how-to/add-output-to-stage)
+  files/directories used by the stage as dependencies, or created as outputs,
```

Comment:
same here - too many details, hard to read - sacrifice some:

`which are already in workspace`
`deps and outs field`
?

Reply:
Please apply my suggestion from commit here too @imhardikj.

Reply:
> deps and outs field

This is a good idea. Something like:

```suggestion
  files/directories are missing from the `deps` or `outs` fields of `dvc.yaml`.
```

It assumes the user will know what it means to be in those fields but I guess you can always click on `dvc.yaml`, find the fields, and see the "dependency" and "output" tooltips from there.

Reply:
Updated this

Reply:
It's better, @imhardikj ! It's always better to go first with a simple statement sentence - what it does. Then explanation if it's needed. This still a very complicates sentence structure and wrong order of sentences within a paragraph to my mind.

Reply:
outdate comment ^^, disregrad